demos/code/code.js

```diff
@@ -549,8 +549,14 @@ Code.initLanguage = function() {
 /**
  * Execute the user's code.
  * Just a quick and dirty eval.  Catch infinite loops.
+ * @param {UIEvent} event Event created from listener bound to the function.
```

Comment:
I think this type is supposed to be `Event`.

Reply:
Changed it to `Event`!